webpack.config.js

```diff
@@ -15,9 +15,12 @@ module.exports = {
       },
       {
         test: /\.css$/i,
-        exclude: /node_modules/,
```

Comment:
My webpack knowledge is limited, why remove this exclude?

Reply:
Our UI library requires that we import it's css at the top level (seen here https://github.com/20BBrown14/digital-terrain-mc-js/pull/4/files#diff-5217b2207602603de708d8fb2097b815R3) and as a result webpack needs to be able to parse it. With the exclude it completely ignores the folder it's in. But now that you mention it I wonder if there's a way to exclude all node_modules but include the one we want. https://webpack.js.org/configuration/module/#ruleinclude
Will give it a go.

Reply:
Yep, that worked. Updated.

Reply:
very nice